src/environments/environment.ts

```diff
@@ -14,3 +14,4 @@ export const environment = {
  * on performance if an error is thrown.
  */
 // import 'zone.js/dist/zone-error';  // Included with Angular CLI.
+export const BASE_URL = 'http://localhost:3000';
```

Comment:
we can use enviroment.prod to get BASE_URL of prod?

Reply:
*Can we use...

But there isn't a BASE_URLon prod yet...

Reply:
You are rigth :(